src/cephadm/cephadm

```diff
@@ -426,10 +426,10 @@ def call(command,  # type: List[str]
         **kwargs
     )
     # get current p.stdout flags, add O_NONBLOCK
-    stdout_flags = fcntl.fcntl(process.stdout, fcntl.F_GETFL)
-    stderr_flags = fcntl.fcntl(process.stderr, fcntl.F_GETFL)
-    fcntl.fcntl(process.stdout, fcntl.F_SETFL, stdout_flags | os.O_NONBLOCK)
-    fcntl.fcntl(process.stderr, fcntl.F_SETFL, stderr_flags | os.O_NONBLOCK)
+    stdout_flags = fcntl.fcntl(process.stdout.fileno(), fcntl.F_GETFL)
+    stderr_flags = fcntl.fcntl(process.stderr.fileno(), fcntl.F_GETFL)
+    fcntl.fcntl(process.stdout.fileno(), fcntl.F_SETFL, stdout_flags | os.O_NONBLOCK)
+    fcntl.fcntl(process.stderr.fileno(), fcntl.F_SETFL, stderr_flags | os.O_NONBLOCK)
```

Comment:
we're not there yet:

```
cephadm:429: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:430: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:431: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:432: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:455: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:465: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
cephadm:475: error: Item "None" of "Optional[IO[Any]]" has no attribute "fileno"
```

Reply:
What about just asserting?
```python
assert process.stdout is not None
assert process.stderr is not None
```
instead of using `.fileno()`?

Reply:
@sebastian-philipp yeah, i think that's the way to go. since this fix does not change the behavior of cephadm, can we merge as it is after "make check" passes? as this practically breaks the "make check" run.